docker_tests/test_docker_compose_quick_start.py

```diff
@@ -134,61 +134,39 @@ def test_trigger_dag_and_wait_for_result(tmp_path_factory, monkeypatch):
     compose_command = ["docker", "compose"]
     success = run_command([*compose_command, "version"], check=False)
     if not success:
-        compose_command = ["docker-compose"]
-        success = run_command([*compose_command, "--version"], check=False)
-        if not success:
-            print("ERROR: Neither `docker compose` nor `docker-compose` is available")
-            sys.exit(1)
+        print("ERROR: `docker compose` not available. Make sure compose plugin is installed")
+        sys.exit(1)
     compose_command.extend(["--project-name", "quick-start"])
     run_command([*compose_command, "config"])
     run_command([*compose_command, "down", "--volumes", "--remove-orphans"])
+    wait_for_containers_timeout = os.getenv("WAIT_FOR_CONTAINERS_TIMEOUT", "500")
+    run_command([*compose_command, "up", "-d", "--wait", "--wait-timeout", wait_for_containers_timeout])
```

Comment:
`--wait-timeout` was added in [Compose v 2.17.0](https://github.com/docker/compose/releases/tag/v2.17.0) but I guess we use 2.14.x in our CI. In additional it also breaks `--timeout` flag until [Compose v 2.17.3](https://github.com/docker/compose/releases/tag/v2.17.3)

Reply:
``console
Good version of Docker: 23.0.0.
Good version of docker-compose: 2.14.1
```